packages/react-composites/src/composites/CallWithChatComposite/CallWithChatControlBar.tsx

```diff
@@ -165,6 +172,18 @@ export const CallWithChatControlBar = (props: CallWithChatControlBarProps & Cont
                   occluding some of its content.
                 */}
               <ControlBar layout="horizontal" styles={centerContainerStyles}>
+                {/*
+                  hidden button to set first tab keypress focus on ControlBar on initial render. 
+                  Unable to set initial focus on Microphone as it causes tooltip to render. On mount, button is
+                  autofocused then immediately hidden
+                */}
+                <ControlBarButton
+                  autoFocus
+                  ariaHidden={true}
+                  data-ui-id={'call-with-chat-autofocus-hidden-button'}
+                  styles={hiddenAutoFocusButtonStyles}
+                  tabIndex={-1}
```

Comment:
could we maybe make a constant for this value? give it a name to say its a tab order remover? maybe: `hiddenFromTabOrderTabIndex` or something?

Reply:
Thanks for the callout but I don't think it is necessary to make it into a constant as it's pretty self explanatory. 
-1 for tabIndex is standard for removing an element from being able to be tabbed into with the keyboard.